internal/engine/podlogmanager.go

```diff
@@ -50,9 +56,18 @@ func (m *PodLogManager) diff(ctx context.Context, st store.RStore) (setup []PodL
 				continue
 			}
 
-			watchByPod, ok := m.watchesByPod[pod.PodID]
-			if ok && watchByPod.cID != pod.ContainerID {
-				watchByPod.cancel()
+			// ASDFG what?!
+			// maybe check for main container id and shut down all if main cID doesn't match?
```

Comment:
this comment block feels more like a WIP comment than one we'd want to have to interpret in a few weeks

Reply:
lol indeed, this is a WIP comment 😅 deleted